### DISCUSSION TITLE
Is ~55% a good successfully aligned rate?

### DISCUSSION THREAD

**Discussion by shouguog** [milaboratory/mixcr#1116] [Q&A] [answered]
I run the command of "mixcr analyze rnaseq-full-length --species hs --keep-non-CDR3-alignments..." and got the following result.

Analysis time: 3.73s
Total sequencing reads: 1751
Successfully aligned reads: 1012 (57.8%)
Alignment failed: no hits (not TCR/IG?): 685 (39.12%)
Alignment failed: low total score: 54 (3.08%)
Overlapped: 300 (17.13%)
Overlapped and aligned: 78 (4.45%)
Overlapped and not aligned: 222 (12.68%)
Alignment-aided overlaps, percent of overlapped and aligned: 13 (16.67%)
No CDR3 parts alignments, percent of successfully aligned: 765 (75.59%)
Partial aligned reads, percent of successfully aligned: 247 (24.41%)
J gene chimeras: 3 (0.17%)
Paired-end alignment conflicts eliminated: 6 (0.34%)
Realigned with forced non-floating bound: 2928 (167.22%)
Realigned with forced non-floating right bound in left read: 0 (0%)
Realigned with forced non-floating left bound in right read: 0 (0%)
TRA chains: 775 (76.58%)
TRA non-functional: 0 (0%)
TRB chains: 229 (22.63%)
TRB non-functional: 0 (0%)
TRAD chains: 8 (0.79%)
TRAD non-functional: 0 (0%)
Trimming report:
  R1 reads trimmed left: 0 (0%)
  R1 reads trimmed right: 0 (0%)
  Average R1 nucleotides trimmed left: 0.0
  Average R1 nucleotides trimmed right: 0.0
======================================

**Comment by PoslavskySV** [comment, marked as answer]
Hi,

It depends on the actual library preparation protocol. For bulk RNA-Seq data a result of few percent is already very good. Could you elaborate more about your data origin (55% looks too high for non-enriched RNA-Seq)?
